accounts/ethstore/src/presale.rs

```diff
@@ -14,7 +14,9 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
 
-use crypto::{self, pbkdf2, Keccak256};
+use crypto::{
+    self, {pbkdf2, Keccak256},
+};
```

Comment:
revert

Reply:
This pull request is still not ready for review, I didn't add anyone to review list. I waited for ipfs/light client to be merged before I come to this and fix it.